docs/quickstart/cross-chain-transfers.md

```diff
@@ -0,0 +1,298 @@
+
+
+Run the export script:
+```sh
+avalanchejs $ ts-node examples/avm/buildExportTx-cchain-avax.ts
+```
+
+This returns:
+```sh
+Success! TXID: Rgg2412kaczRYC3taasvG6bYoqG7tBQG6WfacNdumKDKsVWpF
+```
+
+
+Run the import script:
+```sh
+avalanchejs $ ts-node examples/evm/buildImportTx-xchain.ts
+```
+
+This returns:
+```sh
+Success! TXID: r2yYqcnCJcdeV5gddZ8NUoG5ZD3Ef7DxbkiE9xn4RxFcDdMd1
+```
+
+![Image for post](/img/ajs-getTx.png)
+
+That's it! You've transferred AVAX from the X-Chain to C-Chain! 
+
+
+You can pass this txID `r2yYqcnCJcdeV5gddZ8NUoG5ZD3Ef7DxbkiE9xn4RxFcDdMd1` into [examples/avm/getTx.ts](https://github.com/ava-labs/avalanchejs/blob/master/examples/avm/getTx.ts), plus other similar network settings, then you can run 
+
+```zsh
+avalanchejs $ ts-node examples/avm/getTx.ts
+```
+
+which returns:
+
+```js
+{
+  unsignedTx: {
+    networkID: 1,
+    blockchainID: '2oYMBNV4eNHyqk2fjjV5nVQLDbtmNJzq5s3qs3Lo6ftnC6FByM',
+    outputs: [],
+    inputs: [ [Object] ],
+    memo: '0x',
+    destinationChain: '11111111111111111111111111111111LpoYY',
+    exportedOutputs: [ [Object] ]
+  },
+  credentials: [
+    {
+      fxID: 'spdxUxVJQbX85MGxMHbKw1sHxMnSqJ3QBzDyDYEP3h6TLuxqQ',
+      credential: [Object]
+    }
+  ]
+}
+```
+
+### Transfer from the C-Chain to the X-Chain
+
+To return the AVAX back to the X-Chain, you need to do the transfer in the opposite direction:
+
+* Select the **`examples/evm`** folder to view the AvalancheJS C-Chain examples. To send AVAX from the X-Chain to the C-Chain, select [`evm/buildExportTx-xchain-avax.ts`](https://github.com/ava-labs/avalanchejs/blob/master/examples/evm/buildExportTx-xchain-avax.ts)
+
+* You can change the amount of AVAX to send by editing the _BN_ variable: `avaxAmount`. The sample code assigns this as `1e7` or `10000000` (.01 AVAX)
+
+The fee here will only be for exporting the asset. The import fees will be deducted from the UTXOs present on the Exported Atomic Memory, a memory location where UTXOs lie after getting exported but before being imported. If there is only a single UTXO, then it will be deducted from it.
+
+```ts
+let avaxAmount: BN = new BN(1e7)
+let fee: BN = baseFee.div(new BN(1e9))
+fee = fee.add(new BN(1e6))
+```
+
+:::tip
+When exporting AVAX, be sure to send enough to support import fees (constant .001 AVAX). Sending less than `1e6` or `1000000` (.001 AVAX) may cause the import txn to fail._
+:::
+
+Swap source and destination chains by running the [C-Chain Export](https://github.com/ava-labs/avalanchejs/blob/master/examples/evm/buildExportTx-xchain-avax.ts) and [X-Chain Import](https://github.com/ava-labs/avalanchejs/blob/master/examples/avm/buildImportTx-cchain.ts) scripts.
+
+```zsh
+avalanchejs $ ts-node examples/evm/buildExportTx-xchain-avax.ts
+Success! TXID: UAez3DTv26qmhKKFDvmQTayaXTPAVahHenDKe6xnUMhJbKuxc
+avalanchejs $ ts-node examples/avm/buildImportTx-cchain.ts
```

Comment:
Any changes needed in examples/avm/buildImportTx-cchain.ts before running it?

Reply:
Should be good now

https://github.com/ava-labs/avalanche-docs/blob/328a380d08ef77bf20d3898bcccd8b7ecd991aa8/docs/quickstart/cross-chain-transfers.md?plain=1#L295